api/pkg/webhooks/manager.go

```diff
@@ -56,33 +56,32 @@ func (w *SimpleWebhookManager) InvokeWebhooks(
 		var tmpPayload []byte
 		if client.GetUseDataFrom() == "" {
 			tmpPayload = originalPayload
-		} else if tmpPayload, ok = responsePayloadLookup[client.GetUseDataFrom()]; !ok {
-			// This should only happen if a previous webhook had an error, but did not abort
-			// and the current client is trying to use the response from that client
-			return fmt.Errorf(
-				"webhook name %s not found, this could be because of an error in a previous webhook that this webhook depends on",
-				client.GetUseDataFrom(),
-			)
-		}
-		p, err := client.Invoke(ctx, tmpPayload)
-		if err == nil {
-			responsePayloadLookup[client.GetName()] = p
-			if client.IsFinalResponse() {
-				finalResponse = p
+		} else {
+			tmpPayload, ok = responsePayloadLookup[client.GetUseDataFrom()]
+			if !ok {
+				// NOTE: This should never happen!
```

Comment:
can you elaborate a little, so if i read this comment 6 months later i would understand faster 😅